### PR TITLE
Move Customizing the installation configuration into a separate page

### PR DESCRIPTION
**Pull request by shamsher31**
To fix https://github.com/istio/istio.io/issues/9642

### CODE REVIEW

**Comment by ericvn** [comment]
Waiting on approval from @istio/wg-environments-maintainers.

File: content/en/docs/setup/install/istioctl/index.md [hunk 1]
```diff
@@ -303,345 +303,6 @@ Then run the following `verify-install` command to see if the installation was s
 $ istioctl verify-install -f $HOME/generated-manifest.yaml
 {{< /text >}}
 
-## Customizing the configuration
```
Comment (howardjohn) [inline content/en/docs/setup/install/istioctl/index.md:306]: can we have a link to the other doc here? otherwise most will not find it

Comment (shamsher31) [reply content/en/docs/setup/install/istioctl/index.md:306]: Added. Thanks.

**Comment by shamsher31** [comment]
@ericvn PTAL.

File: content/en/docs/setup/additional-setup/customize-installation/index.md [hunk 2]
```diff
@@ -0,0 +1,355 @@
+---
+title: Customizing the installation configuration
+description: Describes how to customize installation configuration options.
+weight: 55
+aliases:
```
Comment (ericvn) [inline content/en/docs/setup/additional-setup/customize-installation/index.md:5]: I don't think we need an alias here as this is a new page (extracted from a page that still exists).

File: content/en/docs/setup/install/istioctl/index.md [hunk 3]
```diff
@@ -303,344 +303,7 @@ Then run the following `verify-install` command to see if the installation was s
...
-{{< /text >}}
-
-Note that the patches are applied in the given order. Each patch is applied over the output from the
-previous patch. Paths in patches that don't exist in the output manifest will be created.
-
-### List item path selection
-
-Both the `istioctl --set` flag and the `k8s.overlays` field in `IstioOperator` CR support list item selection by `[index]`, `[value]` or by `[key:value]`.
-The --set flag also creates any intermediate nodes in the path that are missing in the resource.
+For advance customization of installation configuration please refer [more guides](/docs/setup/additional-setup/customize-installation/) section.
```
Comment (ericvn) [inline content/en/docs/setup/install/istioctl/index.md:306]: See [Customizing the installation configuration](/docs/setup/additional-setup/customize-installation/) for additional information on customizing the install.

**Review by ericvn — APPROVED** [review]

**Pull request merged**